front/app/containers/ProjectsShowPage/shared/poll/PollMultipleChoice.tsx

```diff
@@ -34,31 +31,25 @@ const MaxText = styled.span`
   font-weight: 400;
 `;
 
-interface InputProps {
-  question: IPollQuestion;
+interface Props {
+  question: IPollQuestionData;
   index: number;
   value: string[] | undefined;
   disabled: boolean;
   onChange: (questionId: string, optionId: string) => () => void;
 }
 
-interface DataProps {
-  options: GetPollOptionsChildProps;
-}
-
-interface Props extends InputProps, DataProps {}
-
 const PollMultipleChoice = ({
   question,
   index,
-  options,
   value,
   disabled,
   onChange,
 }: Props) => {
+  const { data: options } = usePollOptions(question.id);
```

Comment:
```ts
const { data } = usePollOptions(question.id);
const options = data?.options;
```
Maybe in the future we could do this pattern to avoid changing more lines in this component. But again not crucial.

Reply:
Good idea! :))